sdk/arm/storage/2019-06-01/armstorage/README.md

```diff
@@ -0,0 +1,48 @@
+# Azure Storage Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage)
+
+The `armstorage` module provides operations for working with Azure storage resources.
+
+[Source code](https://github.com/Azure/azure-sdk-for-go/tree/master/sdk/arm/storage/2019-06-01/armstorage)
+
+# Getting started
+
+## Install the package
+
+This project uses [Go modules](https://github.com/golang/go/wiki/Modules) for versioning and dependency management.
+
+Install the Azure Storage module:
+
+```sh
+go get -u github.com/Azure/azure-sdk-for-go/sdk/arm/storage/2019-06-01/armstorage
+```
+
+## Prerequisites
+
+- an [Azure subscription](https://azure.microsoft.com/free/)
+- Go 1.13 or above
+
+## Provide Feedback
+
+If you encounter bugs or have suggestions, please
+[open an issue](https://github.com/Azure/azure-sdk-for-go/issues) and assign the `Storage` label.
+
```

Comment:
where's the link to the code samples?

Reply:
It's the `PkgGoDev` link at the top.

Reply:
how about making it more obvious ? 

something like 

# Code Samples

For code samples of how to use this package, please see 
[link to code samples]

Reply:
Will change this up.